### PR TITLE
feat: created products page UI

### PR DESCRIPTION
**Pull request by MahendraDani**
## Related Issue

Closes: #138 

## Description of Changes

- Developed products page UI for the website as per the designs given in the issue.
- The route to the page is `http://localhost:3000/products`.
- The page is fully responsive and dynamic on desktop, tablet and mobile screens.

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots
Please provide any necessary screenshots to illustrate the changes made.

https://github.com/SrijanShovit/carbonops-v2/assets/116940083/4989522d-2b66-4dc7-ab7c-6a9f522cbfe9

### CODE REVIEW

**Review by SrijanShovit — COMMENTED** [review]
Hello MahendraDani, thanks for rising a Pull request, your contribution is valuable to us. The maintainers will review this Pull Request and provide feedback as soon as possible. Keep the great work up!<p></p>

**Comment by MahendraDani** [comment]
@shreya024 If there are any changes please let me know!

File: package-lock.json
Comment (shreya024) [inline package-lock.json:1]: @MahendraDani Resolve the merge conflicts

**Comment by MahendraDani** [comment]
@shreya024 Would you please guide me what conflicts need to be resolved in `package.json`

Because I did not made any changes in `package.json`

**Comment by shreya024** [comment]
> @shreya024 Would you please guide me what conflicts need to be resolved in `package-lock.json`
> 
> Because I did not made any changes in `package-lock.json`

Take a pull

**Comment by MahendraDani** [comment]
@shreya024 I am closing this PR, will raise a fresh PR for this!

**Comment by MahendraDani** [comment]
closing this PR

**Pull request closed without merging**